### ISSUE TITLE
How long did you train your AdaIN? 

### ISSUE THREAD

**Issue by mtroym** [irasin/Pytorch_Adain_from_scratch#2]
How long did you train your AdaIN network and I have another question.

Do you have the problem that the pixel tends gray. Like this.
![image](https://user-images.githubusercontent.com/30402475/76137215-2d8ee100-6075-11ea-9046-463968e87043.png)

**Comment by irasin** [comment]
Hi, @mtroym 

Thank you for your attention.

In fact, I have forgotten how long it will take to train the network, but it will not exceed 12 hours.

The problem that pixel tends gray usually occurs when the input or output of the network have not been normalized or denormalized correctly. Please check the details of the PyTorch vgg19 documentation for more information.  In a simple but effective way,  you can try to train the autoencoder of vgg19(a pretrained vggencoder + a decoder ), which means do not include the adain transformation in the network. If the output still tends to be gray, we can say there must be something wrong in the normalization and denormalization stage of the input and output.

If you have other questions, please let me know. 

Thanks.

**Comment by mtroym** [comment]
@irasin Very thank you for your idea, I found this in Pytorch Official Sites, hope it could help others.
https://pytorch.org/docs/stable/torchvision/models.html
```
# should normalize the tensor after transforms.ToTensor()
normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
```

Thanks! I will try it now.

**Comment by mtroym** [comment]
Maybe I have some problems with my architecture. 
Anyway thank you so much.
